src/chat/takeover.js

```diff
@@ -359,4 +359,12 @@ var takeover = module.exports = function() {
             }
         }
     });
+
+    $('.tse-content').on('click', '.chat-line', function() {
+        if(bttv.settings.get('rmbNameToChat') === false) { return; }
+        var sender = $(this).attr("data-sender");
+        if(typeof sender !== typeof undefined && sender !== false) {
+            $('.ember-chat .chat-interface').find('textarea').val(sender + ", ");
```

Comment:
```data-sender``` providers the normalized username. You want to convert to their display name using the getDisplayName helper.